fe/fe-core/src/main/java/org/apache/doris/nereids/stats/StatsCalculator.java

```diff
@@ -957,7 +957,11 @@ private Statistics computeIntersect(SetOperation setOperation) {
 
     private Statistics computeGenerate(Generate generate) {
         Statistics stats = groupExpression.childStatistics(0);
-        double count = stats.getRowCount() * generate.getGeneratorOutput().size() * 5;
+        int statsFactor = ConnectContext.get().getSessionVariable().generateStatsFactor;
```

Comment:
u could set value checker to session variable in its annotation to ensure it is valid. refer to session variable `queryTimeoutS`